juha/programmers/9/실패율.js

```diff
@@ -0,0 +1,16 @@
+function solution(N, stages) {
+  const answer = [];
+  const result = new Array(N + 1);
+  let total = stages.filter((stage) => stage >= 1).length;
+  for (let i = 1; i <= N; i++) {
+    const failure = stages.filter((stage) => stage === i).length;
+    result[i] = [total ? failure / total : 0, i];
+    total -= failure;
+  }
+  result.shift();
+  result.sort((b, a) => a[0] - b[0]);
+  result.forEach((failureRate) => {
+    answer.push(failureRate[1]);
+  });
```

Comment:
```suggestion
result.sort((b, a) => a[0] - b[0]).forEach((failureRate) => answer.push(failureRate[1]));
```
이 부분은 이런 식으로 한 줄로도 쓸 수 있답니다~~
그리고 forEach 사용에서 위와 같은 상황 시 중괄호를 제외시켜도 무방합니다.

Reply:
훨씬 간결하네요 코드 작성 시 체이닝 최대한 활용하도록 하겠습니다!